tools/sqlite3_api_wrapper/sqlite3_api_wrapper.cpp

```diff
@@ -1323,16 +1343,26 @@ SQLITE_API char *sqlite3_win32_unicode_to_utf8(LPCWSTR zWideText) {
 #endif
 
 // TODO complain
-SQLITE_API void sqlite3_result_blob(sqlite3_context *, const void *, int, void (*)(void *)) {
+SQLITE_API void sqlite3_result_blob(sqlite3_context *context, const void *blob, int n_bytes, void (*)(void *)) {
+	if (blob) {
```

Comment:
We prefer to [return early](https://medium.com/swlh/return-early-pattern-3d18a41bba8):
```cpp
if (!blob) {
   return;
}
```